src/emulator/gui/src/game_selector.cpp

```diff
@@ -170,7 +170,7 @@ void draw_game_selector(HostState &host) {
                 continue;
             if (host.gui.game_selector.icons[game.title_id]) {
                 GLuint texture = host.gui.game_selector.icons[game.title_id].get();
-                ImGui::ImageButton(reinterpret_cast<void *>(texture), ImVec2(icon_size, icon_size));
+                ImGui::Image(reinterpret_cast<void *>(texture), ImVec2(icon_size, icon_size));
```

Comment:
We can't click on icon to launch game anymore. Consider this carefully

Reply:
Click on icon works and launch game, not have probleme for that.

Reply:
Will test it later to confirm

Reply:
already tested and confirmed ^^